src/platform/user/authentication/config/prod.config.js

```diff
@@ -85,4 +86,27 @@ export default {
     oAuthOptions: defaultMobileOAuthOptions,
     externalRedirectUrl: EXTERNAL_REDIRECTS[EXTERNAL_APPS.VA_OCC_MOBILE],
   },
+  [EXTERNAL_APPS.ARP]: {
+    allowedSignInProviders: {
+      idme: true,
+      logingov: true,
+    },
+    allowedSignUpProviders: {
+      idme: true,
+      logingov: true,
+    },
+    isMobile: false,
+    queryParams: {
+      allowOAuth: true,
+      allowPostLogin: true,
+      allowRedirect: false,
+    },
+    oAuthOptions: {
+      ...arpWebOAuthOptions,
+      clientId: 'fe0d4b2cac7935e7eec5946b8ee31643',
```

Comment:
This ID is for staging only, correct? If so, will we need a new ID for production?

Reply:
Also, would it make sense to move this value to a `const`?

Reply:
This is the `prod` clientId. See:
https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/blob/09e2f044cfe4d67a9f2c7ce22606facd18d381ca/apps/vets-api/prod/values.yaml#L2367

Reply:
> Also, would it make sense to move this value to a const?
The current configuration was decided after discussing it with Identity. For context on this, see:
* https://github.com/department-of-veterans-affairs/vets-website/pull/30878#pullrequestreview-2187045731
* https://github.com/department-of-veterans-affairs/vets-website/pull/30878#issuecomment-2239716572
* https://github.com/department-of-veterans-affairs/vets-website/pull/30878#issuecomment-2256505980

Reply:
My idea would be to move this to `src/platform/utilities/oauth/constants.js`, but if this is the agreed-upon approach with Identity then that's fine with me.